### ISSUE TITLE
Clarify requirement "User readiness"

### ISSUE THREAD

**Issue by friedger** [blockstack/app-mining#167]
An app that submits an app for app mining using a different (sub)domain than the domain used to promote the brand of the app should be rejected as not "user-ready".

Example: SpringRole submitted blockstack.springrole.com, however, springrole.com is a clone of blockstack.springrole.com and springrole.com is used in promoting the app, e.g. springroleinc on twitter links to springrole.com. Furthermore, Awario score and PH score is based on springrole.com, not blockstack.springrole.com.

**Comment by jeffdomke** [comment]
@friedger I agree with the spirit of this. Think you're proposing that subdomains should not be used to get around App Mining rules and/or reviewers. Can you please retitle this and also expand a bit on what exact rules you want and which reviewer would enforce this? Or how we should operationalize this test?

**Comment by friedger** [comment]
@jeffdomke It is not limited to subdomains, it should also apply to example-blockstack.com etc.

Rule to be added to the documentation: The app/product's main url must be the same as the url submitted to the app mining program. Main url means the url that is used in communications and marketing about the product by the owner. Rewards are not paid out if this is not the case because the submitted app url is not user-ready.

* Blockstack PBC should check on acceptance of an app that the submitted urls is the url used to promote the app in general

* Each app reviewer should verify that the submitted url is the url they are reviewing.

* Each auditor during the auditing period should verify that Blockstack PBC checked for the above rule.

**Comment by friedger** [comment]
It is at the discretion of Blockstack PBC to decided finally whether the rule is satisfied or not.

**Comment by friedger** [comment]
Most important is that it is stated clearly in the docs

Jeff Domke <notifications@github.com> schrieb am Do., 31. Okt. 2019, 17:54:

**Comment by friedger** [comment]
> That sounds like the kind of thing NIL should test, correct?
For NIL, there is issue #147  that relates to this one.
